### PR TITLE
Added neptune-notebooks recipe

### PR DESCRIPTION
**Pull request by szymon-kuklewicz**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by szymon-kuklewicz** [comment]
@conda-forge-admin, please ping team

File: recipes/neptune-notebooks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  summary: "Neptune's integration with Jupiter and JupiterLab notebooks"
+  doc_url: "https://docs.neptune.ai"
+  dev_url: "https://github.com/neptune-ai/neptune-notebooks"
+
+extra:
+  recipe-maintainers:
+    - szymon-kuklewicz
+    - aniezurawski
+    - hubertjaworskineptune
+    - pitercl
```
Comment (beckermr) [inline recipes/neptune-notebooks/meta.yaml]: These other maintainers need to comment here saying they are ok doing maintenance. 

@aniezurawski @hubertjaworskineptune @pitercl

**Comment by hubertjaworskineptune** [comment]
Hi,
As a neptune developer, I'd like to be a maintainer in this project

**Comment by aniezurawski** [comment]
Hi,
As a neptune developer, I'd like to be a maintainer in this project

**Comment by pitercl** [comment]
Hi,
As a neptune developer, I'd like to be a maintainer in this project

**Pull request merged**